src/main/java/org/kuali/kfs/vnd/businessobject/VendorDetail.java

```diff
@@ -553,8 +558,7 @@ public String getVendorCommoditiesAsString() {
             } else {
                 first = false;
             }
-            // CU customization KFSPTS-3487
-            sb.append(vcc.getCommodityCode().getCommodityDescription());
+            sb.append(vcc.getCommodityCode().getPurchasingCommodityCode());
```

Comment:
Why was this customization removed? Should it be added back in?

Reply:
Good catch, updating